### PR TITLE
Max master new

### PR DESCRIPTION
**Pull request by Maxwell2Gyamfi**
Styled all pages including, patient, staff, create and update
Implemented repeat invoice function

### CODE REVIEW

File: src/main/webapp/appointments/create.jsp [hunk 1]
```diff
@@ -8,104 +8,105 @@
...
+        <link rel="stylesheet" href="${pageContext.request.contextPath}/styles/main.css"/>
+        <title>Patient Dashboard</title>
+    </head>
+    <body>
+        <div class="outer">
+            <div class="header">
+                <div class="innerDashboard">
+                    <img src="${pageContext.request.contextPath}/images/temp_logo.png" alt="logo" id="header-logo">
+                    <ul>
+                        <li><a>${requestScope.user}</a></li>
```
Comment (CharlievWilliams) [inline src/main/webapp/appointments/create.jsp:23]: Did we get this line working correctly now? I remember someone said it was dodgy.

File: src/main/webapp/dashboards/patient.jsp [hunk 2]
```diff
@@ -124,13 +121,13 @@
                                     <td>${appointment.type}</td>
                                     <td>
                                         <form method="get" action="${pageContext.request.contextPath}/appointments/update">
-                                            <button>Edit</button>
+                                            <button id="pinkButtonAlternative">Edit</button>
```
Comment (CharlievWilliams) [inline src/main/webapp/dashboards/patient.jsp:124]: The id will need to be changed to a class in the future. I've already made the change in my own branch so I can do that once this is merged.

File: src/main/webapp/styles/main.css [hunk 3]
```diff
@@ -313,10 +313,10 @@ details[open] summary ~ * {
 
 input[type=date]
 {
-    width:40%;
+    width:42.39%;
```
Comment (CharlievWilliams) [inline src/main/webapp/styles/main.css:316]: This is very specific haha!

File: src/main/webapp/appointments/create.jsp (hunk 1 above)
Comment (Maxwell2Gyamfi) [reply src/main/webapp/appointments/create.jsp:23]: Yes it's working now, displays username

File: src/main/webapp/styles/main.css (hunk 3 above)
Comment (Maxwell2Gyamfi) [reply src/main/webapp/styles/main.css:316]: Haha please don't change, it will break 😹

**Review by Tom-Goring — APPROVED** [review]
LGTM!

**Review by CharlievWilliams — APPROVED** [review]

**Pull request merged**